src/logic/operate.js

```diff
@@ -0,0 +1,21 @@
+import Big from 'big-js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const [firstNum, secondNum] = [Big(numberOne), Big(numberTwo)];
+
+  switch (operation) {
+    case '+':
+      return firstNum.plus(secondNum).toString();
+    case '-':
+      return firstNum.plus(secondNum).toString();
```

Comment:
I guess you wanted to implement the `minus` operation here or up there right?

Reply:
You right !, thanks I will modify this